internal/restorer/restorer.go

```diff
@@ -242,7 +256,10 @@ func (res *Restorer) RestoreTo(ctx context.Context, dst string) error {
 
 			return nil
 		},
-		leaveDir: noop,
+		leaveDir: func(node *restic.Node, target, location string) error {
+			debug.Log("first pass, leaveDir noop %q", location)
```

Comment:
I think it's enough to log the visitor methods which actually do something. Please remove this log call and the one for enterDir in the second pass.

Reply:
Done.